collections/src/main/scala/strawman/collection/mutable/Queue.scala

```diff
@@ -97,14 +97,13 @@ class Queue[A] protected (array: Array[AnyRef], start: Int, end: Int)
     *  @param f   the predicate used for choosing elements
     *  @return
     */
-  def dequeueAll(f: A => Boolean): strawman.collection.Seq[A] = removeHeadWhile(f)
+  def dequeueAll(f: A => Boolean): strawman.collection.immutable.Seq[A] = removeHeadWhile(f)
 
   /** Returns the first element in the queue, or throws an error if there
     *  is no element contained in the queue.
     *
     *  @return the first element.
     */
-  @deprecated("Use Queue.head instead of Queue.front", "2.13.0")
   @`inline` final def front: A = head
```

Comment:
I don’t think it’s a good thing to have several method names to perform the same operations. I’m in favor of keeping the deprecation warning.

Reply:
Adding `Stack.top` convinced me that we should also have `Queue.top`. It's only a matter of convention how the order of the `Seq` is related to the `Queue` or `Stack` semantics. `front` and `top` make it clear what you want in terms of queue or stack.